<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackProcess</class>
 <widget class="QMainWindow" name="BackProcess">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BackProcess</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="factButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>60</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Spustit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sieveButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>180</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Spustit</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="factBar">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>291</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="sieveBar">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>291</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Faktoriál</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Síto</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Výsledek:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Výsledek:</string>
    </property>
   </widget>
   <widget class="QLabel" name="factResult">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>721</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Výsledky</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sieveInput">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="factInput">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="factStop">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>90</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Zastavit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sieveStop">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>210</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Zastavit</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="sieveResult">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>692</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
